docs/_pages/0260-changelog.md

```diff
@@ -22,6 +22,11 @@ next_url: /docs/releases/
 
 ## Recent Updates
 
+* Security update: VSecM Safe has stricter validation routines for its identity.
+* Performance update: VSecM Safe honors SIGTERM and SIGINT signals and gracefully
+  shuts down.
+* Minor updates to the documentation.
```

Comment:
Note to everyone (including myself) @farhan-pasha @abhishek44sharma @ats0stv 

If we make a habit of adding a few lines here before we push a PR, we will spend less time scanning closed issues to write the release notes when the time comes. ;)